src/integrationtests/java/util/IntegrationTestBase.java

```diff
@@ -192,4 +236,18 @@ private void checkResult(final Map<String, JsonNode> checkData) throws IOExcepti
             JSONAssert.assertEquals(expectedJSON.toString(), actualJSON.toString(), false);
         }
     }
+
+    private RabbitTemplate createRabbitMqTemplate() {
+        CachingConnectionFactory cf = new CachingConnectionFactory();
+        cf.setHost(rabbitMqHost);
+        cf.setPort(rabbitMqPort);
+        cf.setUsername(rabbitMqUsername);
+        cf.setPassword(rabbitMqPassword);
+
+        RabbitTemplate rabbitTemplate = new RabbitTemplate(cf);
+        rabbitTemplate.setExchange(exchangeName);
+        rabbitTemplate.setRoutingKey(bindingKey);
+        rabbitTemplate.setQueue(consumerName);
```

Comment:
We agreed that you should not send to a specific queue. but only to an exchange. Please remove set queue.

Reply:
I think i missunderstood, will remove :)